src/client/components/pages/NonClassMeetings/NonClassMeetingsTable.tsx

```diff
@@ -0,0 +1,210 @@
+      </TableRow>
+    </TableHead>
+    <TableBody>
+      {
+        nonClassMeetingList.map(({
+          id,
+          area,
+          title,
+          expectedSize,
+          notes,
+          contactEmail,
+          contactName,
+          contactPhone,
+          spring,
+          fall,
+        }, index) => (
+          <TableRow isStriped={index % 2 !== 0} key={id}>
+            <TableCell backgroundColor={getAreaColor(area)}>{area}</TableCell>
+            <TableCell>{title}</TableCell>
+            <TableCell>{expectedSize}</TableCell>
+            <TableCell>{notes}</TableCell>
+            <TableCell>
+              {
+                /*
+                  * This is being used instead of just outputting the fields
+                  * directly as a way to filter out any null values. This
+                  * ensures that if a phone number, name or email is missing
+                  * we don't end up with blank lines where the "<br />"s still
+                  * are, but with no data before them.
+                  *
+                  * The same result could be accomplished with lots of ternary
+                  * logic, but this felt like a cleaner (and more scalable) way
+                  * to solve the problem.
+                  */
+                [
+                  contactName,
+                  contactEmail,
+                  contactPhone,
+                ]
+                  .filter((info) => !!info)
+                  .map((info) => (<p key={info}>{info}</p>))
+              }
+            </TableCell>
+            <TableCell>
+              <CellLayout>
+                <TableCellList>
+                  {fall.meetings.map(({
+                    id: meetingId,
+                    day,
+                    startTime,
+                    endTime,
+                    room,
+                  }) => (
+                    <TableCellListItem key={meetingId}>
+                      <MeetingGrid>
+                        <MeetingGridSection area="time">
+                          <div>{dayEnumToString(day)}</div>
+                          <div>{`${startTime}-${endTime}`}</div>
+                        </MeetingGridSection>
+                        {room && (
+                          <>
+                            <MeetingGridSection area="room">
+                              {room.name}
+                            </MeetingGridSection>
+                            <MeetingGridSection area="campus">
+                              <CampusIcon>{room.campus}</CampusIcon>
+                            </MeetingGridSection>
+                          </>
+                        )}
+                      </MeetingGrid>
+                    </TableCellListItem>
+                  ))}
+                </TableCellList>
+                <BorderlessButton
+                  id={`${id}-${spring.id}-edit-meetings-button`}
+                  onClick={() => {}}
+                  variant={VARIANT.INFO}
+                >
```

Comment:
It seems like our convention going forward is to add `alt` text to each of the buttons for accessibility.

Reply:
Yes! Good point.

Sidenote - I might raise an issue on `mark-one` to make that a required prop going forward.

Reply:
I've opened https://github.com/seas-computing/mark-one/issues/111 for this

Reply:
I have now added this in 71fc9fd and 77102a02a2b53b8198d0abcfa5ed2f3026a224df